src/module.c

```diff
@@ -1158,6 +1158,26 @@ void swapDbEventCallback(RedisModuleCtx *ctx, RedisModuleEvent e, uint64_t sub,
     }
 }
 
+void keyAddedToDbDict(RedisModuleCtx *ctx,
+                      RedisModuleString *key,
+                      void *value,
+                      int swap_key_metadata) {
+    Series *series = (Series *)value;
+    series->in_ram = true;
+}
+
+int keyRemovedFromDbDict(RedisModuleCtx *ctx,
+                         RedisModuleString *key,
+                         void *value,
+                         int swap_key_metadata,
+                         int writing_to_swap) {
+    Series *series = (Series *)value;
+    if (!!writing_to_swap) {
```

Comment:
Can you add a comment what this variable means and why we need to change the flag only if this variable is true?